enrichment_wrangler.py

```diff
@@ -124,23 +113,21 @@ def lambda_handler(event, context):
         if not json_response["success"]:
             raise exception_classes.MethodFailure(json_response["error"])
 
-        aws_functions.save_data(bucket_name, out_file_name, json_response["data"],
-                                sqs_queue_url, outgoing_message_group_id, location)
+        aws_functions.save_to_s3(bucket_name, out_file_name, json_response["data"])
 
         logger.info("Successfully sent data to s3.")
 
         anomalies = json_response["anomalies"]
 
         if anomalies != "[]":
-            aws_functions.save_to_s3(bucket_name, "Anomalies", anomalies, location)
+            aws_functions.save_to_s3(bucket_name, "Anomalies", anomalies)
```

Comment:
does this function have a default for location, or is this going to break it?

Reply:
Yep has a default of "" so it's fine.